libclingo/clingo/clingocontrol.hh

```diff
@@ -422,6 +423,14 @@ public:
     Int64Vec optimization() const override {
         return model_->costs ? Int64Vec(model_->costs->begin(), model_->costs->end()) : Int64Vec();
     }
+    std::vector<Potassco::Weight_t> priorities() const override {
```

Comment:
Looks ok to me. However, I would suggest accessing the minimize constraint via the Enumerator that is referenced in the model instead of the shared context. E.g. something like:
```cpp
if (const auto* mini = model_->ctx ? model_->ctx->minimizer() : nullptr; mini) {
   ret.assign(mini->prios.begin(), mini->prios.end());
}
```